### PR TITLE
python sol for greg and array

### PR DESCRIPTION
**Pull request by freakin23**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [ ] I have linked this PR to any issues that it closes.

### CODE REVIEW

File: content/3_Silver/More_Prefix_Sums.mdx [hunk 1]
```diff
@@ -725,6 +725,35 @@ int main() {
 ```
 
 </CPPSection>
+
+<PySection>
+```py
+n, m, k = map(int, input().split())
+arr = list(map(int, input().split()))
+
```
[the remark this answers is not in the document]
Comment (freakin23) [reply content/3_Silver/More_Prefix_Sums.mdx:733]: yes

[the remark this answers is not in the document]
Comment (freakin23) [reply content/3_Silver/More_Prefix_Sums.mdx:733]: done

[the remark this answers is not in the document]
Comment (freakin23) [reply content/3_Silver/More_Prefix_Sums.mdx:733]: can i ask you to go through the kosaraju's  pr ?

[the remark this answers is not in the document]
Comment (SansPapyrus683) [reply content/3_Silver/More_Prefix_Sums.mdx:733]: quick question about that- some people think that editorial was also ai written :skull:

[the remark this answers is not in the document]
Comment (freakin23) [reply content/3_Silver/More_Prefix_Sums.mdx:733]: To be honest it's not. I have gone through several blogs and few books to write that.

**Comment by ryanchou-dev** [comment]
send ac screenshot?

**Comment by freakin23** [comment]
> send ac screenshot?

Screenshot from 2024-08-17 13-03-23

**Review by SansPapyrus683 — APPROVED** [review]

**Pull request merged**